.cloudbees/staging/action.yml

```diff
@@ -0,0 +1,109 @@
+apiVersion: automation.cloudbees.io/v1alpha1
```

Comment:
This repo is public - right?   Not sure if it good idea to have environment specific  internal folder "staging" there?

Reply:
it's the only way to self-test without requiring a follow-up change that cannot be tested

Reply:
Ok fine with me.